protos/core.proto

```diff
@@ -22,6 +22,7 @@ message RunConfig {
   bool include_dependencies = 3;
   bool full_refresh = 2;
   bool carry_on = 4;
+  repeated string tags = 5;
```

Comment:
would you mind also adding this to the relevant section of `schedules.proto`? (no code should need to change, just the protobuf)

Reply:
Sure